### DISCUSSION TITLE
Ading B-Hyve Integration

### DISCUSSION THREAD

**Discussion by rvanscherpe** [sebr/bhyve-home-assistant#253] [General]
Hi. 
I'm trying to add the integration into HA .  Its not listed in the HA integrations so I tried adding it as a repository but it comes back not a valid add-on repository. I'm using https://github.com/sebr/bhyve-home-assistant.git

Ron

**Comment by sebr** [comment]
You need to install it via HACS. There are detailed instructions in the Readme
